README.md

```diff
@@ -31,165 +31,202 @@
+# ⭐️ Premium Features
+
+**Leverage the Quality Assurance best practices**: The Giskard catalog enables you to easily contribute and load data slicing & transformation functions such as AI-based detectors (toxicity, hate, etc.), generators (typos, paraphraser, etc.), or evaluators.
+
+<p align="center">
+  <img src="readme/catalog_example.png" alt="Scan Example" width="700px">
+</p>
+
+## Start the Giskard server
+
+For full functionality start **Giskard server** with: 
+```sh
+giskard server start
+```
+
+🚀 That's it! Access at http://localhost:19000
+
+## Upload your test suite to the Giskard server
+
+You can then **upload the test suite** to the local Giskard server. This will enable you to:
 - Compare the quality of different models to decide which one to promote
 - Debug your tests to diagnose the identified issues
 - Create more domain-specific tests relevant to your use case
 - Share results, and collaborate with your team to integrate business feedback
 
-First, install the Giskard server by following [this documentation](https://docs.giskard.ai/en/latest/guides/installation_app/index.html)
+1. First, make sure Giskard server is installed 
+    <details>
+      <summary>How to check if the Giskard server is running</summary>
+      
+      - check if http://localhost:19000 is running
+      - or use `giskard server status`
+    </details>
 
-```python
-# Create a Giskard client after having installed the Giskard server (see documentation)
-token = "API_TOKEN"  # Find it in Settings in the Giskard server
-client = GiskardClient(
-    url="http://localhost:19000", token=token  # URL of your Giskard instance
-)
+2. Then execute the ML worker in your notebook:
+    ```python
+       !giskard worker start -d -k YOUR_TOKEN
+    ```
 
-my_project = client.create_project("my_project", "PROJECT_NAME", "DESCRIPTION")
 
-# Upload to the current project
-test_suite.upload(client, "my_project")
 
-```
+
+3. Finally upload test suite to the giskard server using the following code:
+    ```python
+    token = "API_TOKEN"  # Find it in Settings in the Giskard server
+    client = GiskardClient(
+        url="http://localhost:19000", token=token  # URL of your Giskard instance
+    )
+    
+    my_project = client.create_project("my_project", "PROJECT_NAME", "DESCRIPTION")
     
-For more information on uploading to your local Giskard server, go to the [Upload an object to the Giskard server](https://docs.giskard.ai/en/latest/guides/upload/index.html) page.
+    # Upload to the current project
+    test_suite.upload(client, "my_project")
+    
+    ```
+
 
-## How to contribute
+# 👋 Community
 We welcome contributions from the Machine Learning community!
 
 Read this [guide](CONTRIBUTING.md) to get started.
 
-<br />
-
-## Like what we're doing?
-
 🌟 [Leave us a star](https://github.com/Giskard-AI/giskard), it helps the project to get discovered by others and keeps us motivated to build awesome open-source tools! 🌟
```

Comment:
before the leave us a star section there should be a "Join our thriving community ... " with a link to our discord